### PR TITLE
release notes for python form recognizer

### PR DESCRIPTION
**Pull request by kristapratico**
Following what looks like new guidance for release notes:
https://azure.github.io/azure-sdk/policies_releasenotes.html#whats-a-developer-impacting-change

### CODE REVIEW

File: releases/2020-06/python.md [hunk 1]
```diff
@@ -44,6 +46,22 @@ If you have a bug or feature request for one of the libraries, please post an is
 - We are now targeting the service's v3.0 API, instead of the v3.0-preview.1 API
 - Updated the models to correspond with service changes
 
+### Form Recognizer [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md)
```
Comment (iscai-msft) [inline releases/2020-06/python.md:49]: I think this should point to b3's changelog

File: releases/2020-06/python.md [hunk 2]
```diff
@@ -44,6 +46,22 @@ If you have a bug or feature request for one of the libraries, please post an is
...
+- All asynchronous long running operation methods now return an instance of an AsyncLROPoller from azure-core
+- All asynchronous long running operation methods have been renamed with the `begin_` prefix to indicate that they return a poller object
+- Method `get_form_training_client()` is removed from `FormRecognizerClient` and `get_form_recognizer_client()` is added to `FormTrainingClient`
+- Other method and property renaming detailed in changelog
+
+#### New Features
+
+- Support to copy a custom model from one Form Recognizer resource to another
+- Authentication using azure-identity credentials now supported
+- All long running operation methods now accept the keyword argument continuation_token to restart the poller from a saved state
```
Comment (iscai-msft) [inline releases/2020-06/python.md:62]: Would put continuation_token in ``

File: releases/2020-06/python.md [hunk 3]
```diff
@@ -44,6 +46,22 @@ If you have a bug or feature request for one of the libraries, please post an is
 - We are now targeting the service's v3.0 API, instead of the v3.0-preview.1 API
 - Updated the models to correspond with service changes
 
+### Form Recognizer [Changelog](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/formrecognizer/azure-ai-formrecognizer/CHANGELOG.md)
+
+#### Breaking changes
+
+- All asynchronous long running operation methods now return an instance of an AsyncLROPoller from azure-core
+- All asynchronous long running operation methods have been renamed with the `begin_` prefix to indicate that they return a poller object
+- Method `get_form_training_client()` is removed from `FormRecognizerClient` and `get_form_recognizer_client()` is added to `FormTrainingClient`
```
Comment (iscai-msft) [inline releases/2020-06/python.md:55]: I feel like the changelog could do withoutt line 55, but don't have strong feelings

File: releases/2020-06/python.md (hunk 1 above)
Comment (kristapratico) [reply releases/2020-06/python.md:49]: Can't do this until the release date is updated on changelog. otherwise link won't be valid

Comment (iscai-msft) [reply releases/2020-06/python.md:49]: oh, didn't think about that. I'm good with keeping it like this then. Thanks!

Comment (kristapratico) [reply releases/2020-06/python.md]: update to b3 link now that I have the date

**Pull request merged**